SCF-Session-5/N_Queens/NQueenProblem.java

```diff
@@ -0,0 +1,102 @@
+package N_Queens;
+
+/**
+ *this class is used to check whether queen's problem can be solved for the given dimension
+ * @author Ishita_Mathur
+ *
+ */
+public class NQueenProblem 
+{
+	/**
+     	* this method checks whether solution exists or not
+     	* @param N dimension of matrix
+     	* @return true if solution exist, false otherwise
+     	*/
+	public boolean nQueens(int N)
+    	{
+        	int row = 0; 
+        	int[][] board = new int[N][N];
+        
+        	// initializing the board with 0 at all places
+        	for(int i = 0; i < N; i++)
+            		for(int j = 0; j < N; j++)
+                		board[i][j] = 0;
+        
+        	boolean result = placeQueen(N, board, row);
```

Comment:
Variables should be declared at top of the any block. Statement can be written as `return placeQueen(N, board, row);`.

Reply:
done